engine/apps/alerts/models/alert.py

```diff
@@ -240,30 +279,31 @@ def render_group_data(cls, alert_receive_channel, raw_request_data, is_demo=Fals
         if group_distinction is not None:
             group_distinction = hashlib.md5(str(group_distinction).encode()).hexdigest()
 
-        if resolve_condition_template is not None:
-            try:
-                is_resolve_signal = apply_jinja_template(resolve_condition_template, payload=raw_request_data)
-            except (JinjaTemplateError, JinjaTemplateWarning) as e:
-                logger.warning(
-                    f"resolve_condition_template error on channel={alert_receive_channel.public_primary_key}: {e.fallback_message}"
-                )
-
-            if isinstance(is_resolve_signal, str):
-                is_resolve_signal = is_resolve_signal.strip().lower() in ["1", "true", "ok"]
-            else:
-                is_resolve_signal = False
-        if acknowledge_condition_template is not None:
-            try:
-                is_acknowledge_signal = apply_jinja_template(acknowledge_condition_template, payload=raw_request_data)
-            except (JinjaTemplateError, JinjaTemplateWarning) as e:
-                logger.warning(
-                    f"acknowledge_condition_template error on channel={alert_receive_channel.public_primary_key}: {e.fallback_message}"
-                )
-
-            if isinstance(is_acknowledge_signal, str):
-                is_acknowledge_signal = is_acknowledge_signal.strip().lower() in ["1", "true", "ok"]
-            else:
-                is_acknowledge_signal = False
```

Comment:
this was simply deduplicating a lot of copy/paste into `Alert._apply_jinja_template_to_alert_payload_and_labels`